week09_policy_II/ppo.ipynb

```diff
@@ -265,11 +265,11 @@
     "In it you should add two keys to the trajectory: `\"advantages\"` and `\"value_targets\"`. In GAE the advantages\n",
     "$A_t^{\\mathrm{GAE}(\\gamma,\\lambda)}$ are essentially defined as the exponential \n",
     "moving average with parameter $\\lambda$ of the regular advantages \n",
-    "$\\hat{A}^{(n)}(s_t) = \\sum_{l=0}^{T-1} \\gamma^l r_{t+l} + \\gamma^{T} V^\\pi(s_{t+l}) - V^\\pi(s_t)$. \n",
+    "$\\hat{A}^{(T)}(s_t) = \\sum_{l=0}^{T-1} \\gamma^l r_{t+l} + \\gamma^{T} V^\\pi(s_{T}) - V^\\pi(s_t)$. \n",
```

Comment:
There are conflicts with the base branch because of another PR that modified this file and has already been merged. Can you rebase on `spring20` and force-push?

Alternatively, you can just copy the version of this file that is already in `spring20` into this PR. This will ensure that there are no conflicts.

Reply:
I have pushed changes by:
bed73ac6972443006982d7a60f9cbfab1830f3ba

Is it OK?

Reply:
Well, there is still a merge conflict =) Can you address it?